rsearch/config.ini

```diff
@@ -0,0 +1,18 @@
+; comment
+[api]
+url=http://192.168.0.10:8080
```

Comment:
Couldn't this just be `watchURL` as a single item? Does it actually vary between deployments?